// sanity/lib/structure.js
'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});
var structure = require('./_chunks/structure-AbBdhb-P.js');
require('react/jsx-runtime');
require('@sanity/icons');
require('@sanity/ui');
require('react');
require('./_chunks/TooltipDelayGroupProvider-IMd1Peab.js');
require('sanity');
require('@sanity/telemetry/react');
require('lodash/template.js');
require('lodash/isEqual.js');
require('sanity/router');
require('react-rx');
require('rxjs');
require('@sanity/types');
require('@sanity/util/paths');
require('lodash/omit.js');
require('lodash/isNumber.js');
require('lodash/isString.js');
require('@sanity/uuid');
require('lodash/sortBy.js');
require('lodash/findIndex.js');
exports.CommentsIntentProvider = structure.CommentsIntentProvider;
exports.ComponentBuilder = structure.ComponentBuilder;
exports.ComponentViewBuilder = structure.ComponentViewBuilder;
exports.ConfirmDeleteDialog = structure.ConfirmDeleteDialogContainer;
exports.DEFAULT_INTENT_HANDLER = structure.DEFAULT_INTENT_HANDLER;
exports.DocumentBuilder = structure.DocumentBuilder;
exports.DocumentInspectorHeader = structure.DocumentInspectorHeader;
exports.DocumentListBuilder = structure.DocumentListBuilder;
exports.DocumentListItemBuilder = structure.DocumentListItemBuilder;
exports.DocumentListPane = structure.DocumentListPane;
exports.DocumentPane = structure.DocumentPane;
exports.DocumentPaneProvider = structure.DocumentPaneProvider;
exports.DocumentTypeListBuilder = structure.DocumentTypeListBuilder;
exports.FormViewBuilder = structure.FormViewBuilder;
exports.GenericListBuilder = structure.GenericListBuilder;
exports.GenericViewBuilder = structure.GenericViewBuilder;
exports.HELP_URL = structure.HELP_URL;
exports.InitialValueTemplateItemBuilder = structure.InitialValueTemplateItemBuilder;
exports.ListBuilder = structure.ListBuilder;
exports.ListItemBuilder = structure.ListItemBuilder;
exports.MenuItemBuilder = structure.MenuItemBuilder;
exports.MenuItemGroupBuilder = structure.MenuItemGroupBuilder;
exports.PaneLayout = structure.PaneLayout;
exports.PaneRouterContext = structure.PaneRouterContext;
exports.SerializeError = structure.SerializeError;
exports.StructureToolProvider = structure.StructureToolProvider;
exports.component = structure.component;
exports.createStructureBuilder = structure.createStructureBuilder;
exports.defaultInitialValueTemplateItems = structure.defaultInitialValueTemplateItems;
exports.defaultIntentChecker = structure.defaultIntentChecker;
exports.documentFromEditor = structure.documentFromEditor;
exports.documentFromEditorWithInitialValue = structure.documentFromEditorWithInitialValue;
exports.form = structure.form;
exports.getOrderingMenuItem = structure.getOrderingMenuItem;
exports.getOrderingMenuItemsForSchemaType = structure.getOrderingMenuItemsForSchemaType;
exports.getTypeNamesFromFilter = structure.getTypeNamesFromFilter;
exports.isDocumentListItem = structure.isDocumentListItem;
exports.maybeSerializeInitialValueTemplateItem = structure.maybeSerializeInitialValueTemplateItem;
exports.maybeSerializeMenuItem = structure.maybeSerializeMenuItem;
exports.maybeSerializeMenuItemGroup = structure.maybeSerializeMenuItemGroup;
exports.maybeSerializeView = structure.maybeSerializeView;
exports.menuItemsFromInitialValueTemplateItems = structure.menuItemsFromInitialValueTemplateItems;
exports.shallowIntentChecker = structure.shallowIntentChecker;
exports.structureLocaleNamespace = structure.structureLocaleNamespace;
exports.structureTool = structure.structureTool;
exports.useDocumentPane = structure.useDocumentPane;
exports.useDocumentTitle = structure.useDocumentTitle;
exports.usePaneRouter = structure.usePaneRouter;
exports.useStructureTool = structure.useStructureTool;
